### DISCUSSION TITLE
viewing referrer logs

### DISCUSSION THREAD

**Discussion by dcsan** [vercel/community#1957] [Help] [answered]
### Summary

I'd like to see where my traffic is coming from to a vercel hosted site. 
I can't find any references to referrer tracking 

like old school days looking through apache logs...

is this info available? through the analytics (which we're not using currently) ?

**Comment by TooTallNate** [comment, marked as answer]
You can enable [Vercel Audiences](https://vercel.com/docs/concepts/analytics/audiences) on your project to get Referrer statistics.

**Reply by dcsan** [reply]
that seems to be the domains, do you know if you can drill down to actual URLs? 

I did read about the drilldown back for t.co links but it basically just links to twitter's search with your domain... 
https://vercel.com/docs/concepts/analytics/audiences/filtering#find-tweets-from-t.co-referrer

**Reply by TooTallNate** [reply]
You can drill down specific URL for sites that enable it, but it will depend on the site's [Referrer-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy). The default is `strict-origin-when-cross-origin` which does not send the path/query information cross-origin:

> Send the origin, path, and querystring when performing a same-origin request. For cross-origin requests send the origin (only) when the protocol security level stays same (HTTPS→HTTPS). Don't send the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) header to less secure destinations (HTTPS→HTTP).
